src/layer/vector/Canvas.js

```diff
@@ -138,6 +138,13 @@ export var Canvas = Renderer.extend({
 	_initPath: function (layer) {
 		this._updateDashArray(layer);
 		this._layers[Util.stamp(layer)] = layer;
+		if (layer.options.keyboard) {
+			layer._path = DomUtil.create('div', 'leaflet-canvas-shadow-path', this._container);
```

Comment:
I think we should avoid the term "shadow", it sounds like Shadow DOM, or could perhaps be confused with marker shadows.

Reply:
Agree, changed the class name to avoid confusion